src/libslic3r/PrintConfig.cpp

```diff
@@ -4317,6 +4317,28 @@ void PrintConfigDef::init_fff_params()
     def->mode = comAdvanced;
     def->set_default_value(new ConfigOptionFloatOrPercent(200, true));
 
+    def = this->add("spiral_starting_flow_rate", coPercent);
```

Comment:
Should we use a float option instead to match other flow ratio settings like `bridge_flow`?